storage-proofs/porep/src/stacked/vanilla/cache.rs

```diff
@@ -0,0 +1,15 @@
+use std::env;
+use std::path::{Path, PathBuf};
+
+pub const SDR_CACHE_ENV_VAR: &str = "FIL_PROOFS_SDR_CACHE";
+pub const PARAMETER_CACHE_DIR: &str = "/var/tmp/filecoin-proof-sdr-cache/";
+
+pub fn dir_name() -> PathBuf {
+    env::var(SDR_CACHE_ENV_VAR)
+        .map(PathBuf::from)
+        .unwrap_or(PathBuf::from(PARAMETER_CACHE_DIR))
+}
```

Comment:
For env var configuration, please use the mechanism in `settinsgs.rs` for consistency and to consolidate in one place developers and users can keep track of.